js/map_utils.js

```diff
@@ -264,9 +285,38 @@ function showToolTip(d, category) {
   }, toolTipDelay);
 }
 
-function hideToolTip() {
-  d3.select(".tooltip")
-    .classed("hidden", true);
+function clearLegendText() {
+
+  waterUseViz.elements
+    .buttonBox
+    .selectAll("#legend-title")
+    .text("Water Use");
+
+  waterUseViz.elements
+    .buttonBox
+    .selectAll("#total_button_text")
+    .text("");
+    
+  waterUseViz.elements
+    .buttonBox
+    .selectAll("#irrigation_button_text")
+    .text("");
+    
+  waterUseViz.elements
+    .buttonBox
+    .selectAll("#industrial_button_text")
+    .text("");
+    
+  waterUseViz.elements
+    .buttonBox
+    .selectAll("#thermoelectric_button_text")
+    .text("");
+    
+  waterUseViz.elements
+    .buttonBox
+    .selectAll("#publicsupply_button_text")
+    .text("");
```

Comment:
Here I see that it's useful to set all of the text elements to make sure they're all empty, but there's still a way to replace these 24 lines with 5 lines:
```js
waterUseViz.elements
  .buttonBox
  .selectAll('.category-amount')
  .text("");
```
because `.text` is getting called after a `.selectAll`, it automatically gets looped over all the elements in the selection (in this case all 5 of the text boxes with class `category-amount`)

Reply:
Doh! Yeah, I should have thought of that